includes/apple-exporter/class-exporter.php

```diff
@@ -204,7 +204,7 @@ private function generate_json() {
 		$json = array(
 			'version'    => '1.1',
 			'identifier' => 'post-' . $this->content_id(),
-			'language'   => 'en',
+			'language'   => get_bloginfo( 'language' ),
```

Comment:
Great idea, I hadn't considered that. Even though it's English-language only for now, this change should at least mean the JSON uses a more specific en-US, en-GB or en-AU code — and future-proof it too!